src/assets/css/style.css

```diff
@@ -1,3 +1,20 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+.service-card-gradient-color {
+    background: linear-gradient(
+        90deg,
+        #f5f8fc 60%,
+        rgba(245, 248, 252, 0) 100%
+    );
+    transition: all linear 0.3s;
+}
+
+.service-card-gradient-color:hover {
+    background: linear-gradient(
+        90deg,
+        #dee2ee 60%,
+        rgba(222, 226, 238, 0) 100%
+    );
+}
```

Comment:
I first implemented it with tailwindcss.it works but not for hover state.I will check again maybe it works :)

Reply:
It can be implemented by Tailwind. But if you are aware of the transition you need to change `background-position` instead of different gradient.

This is an [example](https://stackoverflow.com/a/67853821/4038528).